README.md

```diff
@@ -1,138 +1,23 @@
 # Microservices March Demo Architecture Platform (WIP)
-
 Shared infrastructure for the Microservices March demo architecture.
 
 ## Usage
+The aim is to set things up as manually as possible without involving something heavy like Kubernetes. Future iterations might have examples of more advanced setups.
 
-The aim is to set things up as manually as possible without creating "magic" via Docker Compose or involving something heavy like Kubernetes.  Future iterations might have examples of more advanced setups.
-
-### Prerequisites
-
-Make sure you have this repo as well as the following additional repos checked out:
-
-* [notifier](https://github.com/microservices-march/notifier)
-* [messenger](https://github.com/microservices-march/messenger)
-
-### Start the Shared Platform Infrastructure
-
-From this repository, run:
-
-```bash
-docker-compose up -d
-```
-
-### Start the `messenger` Service
-
-1. From the `messenger` repository, build the Docker image:
-
-    ```bash
-    docker build -t messenger .
-    ```
-
-2. From the `messenger` repository, start the PostgreSQL database:
-
-    ```bash
-    docker-compose up -d
-    ```
-
-3. Start the `messenger` service in a container:
-
-    ```bash
-    docker run -d -p 8083:8083 --name messenger -e PGPASSWORD=postgres -e CREATE_DB_NAME=messenger -e PGHOST=messenger-db-1 -e AMQPHOST=rabbitmq -e AMQPPORT=5672 -e PORT=8083 --network mm_2023 messenger
-    ```
-
-4. SSH into the container to set up the PostgreSQL DB:
-
-    ```bash
-    docker exec -it messenger /bin/bash
-    ```
-
-5. Create the PostgreSQL DB:
-
-    ```bash
-    PGDATABASE=postgres node bin/create-db.mjs
-    ```
-
-6. Create the PostgreSQL DB tables:
-
-    ```bash
-    node bin/create-schema.mjs
-    ```
-
-7. Create some PostgreSQL DB seed data:
-
-    ```bash
-    node bin/create-seed-data.mjs
-    ```
+## Running just shared infrastructure
+If you are running some subset of the services manually, just run `docker-compose up` from this repository to start up the message queue.  This does not set up the NGINX load balancer.
 
-### Start the `notifier` Service
+## Setting up the whole demo architecture.
+Starting up the entire system including the applications is not part of the role of this repository. However, instructions are included here for simplicity.
 
-1. From the `notifier` repository, build the Docker image:
+We provide two methods to set things up:
+1. [Quick Setup](docs/quick-setup.md): Based on `docker-compose`. Use this one if you just want to get started quickly.
+1. [Manual Setup](docs/manual-setup.md): Uses some `docker-compose` but mostly raw docker commands. Use this if you want go through what the quick start does for you automatically for learning.
```

Comment:
```suggestion

1. [Manual Setup](https://github.com/microservices-march/platform/blob/main/docs/manual-setup.md): Follow this guide. if you want to manually go through most steps. Recommended for learning. 
2. [Quick Setup](https://github.com/microservices-march/platform/blob/main/docs/quick-setup.md): Follow this guide if you want to get started quickly. Automates mosts steps covered in the Manual Setup guide.

```
Let's put the manual setup first to try to help guide users towards it. There is no need to specify that we are using Docker Compose to various extents. The respective guide descriptions do that already.

Reply:
Good suggestion, thanks!